src/controllers/requestController.ts

```diff
@@ -76,6 +78,13 @@ export class RequestController {
     }
 
     private async runCore(httpRequest: HttpRequest) {
+        if (undefined !== httpRequest.confirmSendMsg) {
+            const userConfirmed = await window.showQuickPick(['Yes', 'No'], { canPickMany: false, placeHolder: httpRequest.confirmSendMsg });
+            if ('Yes' !== userConfirmed) {
+              return;
+            }
+        }

```

Comment:
I think we should move this logic to the `run` method, since `rerun` method doesn't need to rely on this logic personally.

Reply:
I'll leave this as a setting "confirmOnRerun"